translib/test/sflow/README.md

```diff
@@ -0,0 +1,9 @@
+# Gnmi Testing
+
+Following are the instructions on how to build and execute transformer gnmi unit test.
```

Comment:
Can we avoid gnmi specific test steps in sonic-mgmt-common repo?
I think these are manual test cases that require manual verification. There are gotest cases in sonic-gnmi repo, which bring up a test server and run get/set tests by reading the input from files. It is better to move these stuffs there. If you cannot enhance the sonic-gnmi test suite now, please attach these to the PR for future reference instead of committing to sonic-mgmt-common

Reply:
We have removed the manual testing changes